integrations/postgres-integration.md

```diff
@@ -0,0 +1,130 @@
+# Kubecost / Postgres Integration
+
+{% hint style="info" %}
+This feature is only supported for Kubecost Enterprise.
+{% endhint %}
+
+Often times, users prefer to gather all their data and metrics in a single data warehouse. From there, they can run queries and generate reports to analyze the data across multiple sources. The Postgres Integration feature enables users to periodically query and export their Kubecost data to a central Postgres database.
+
+## Example use cases
+
+- Customized dashboarding used by FinOps/Analytics teams.
+- Single view populated by multiple data sources, to be used by developer/finance/operations teams.
+- Customized budgeting and forecasting via an in-house data pipeline.
+
+## Usage
+
+Prerequisites:
+
+- A running Postgres instance that is reachable by Kubecost.
```

Comment:
I think we have to call out a requirement to create a user with the following permissions:
CREATE, INSERT, UPDATE

I'm not sure if we think the GRANT CREATE will cause any turbulence but if we keep the CREATE IF NOT EXISTS line in the integration it is a requirement.

Reply:
Good catch. I've just added in a line describing this.

Reply:
This may be out of scope for this PR, but what I'd ultimately like to see is maybe a quick start guide for this integration (strictly called out as non-production grade) using something like [Zalando's postgres operator](https://github.com/zalando/postgres-operator).

Reply:
Sure, that also works.